src/webportal/src/app/job/job-view/fabric/job-detail/components/summary.jsx

```diff
@@ -366,6 +366,15 @@ export default class Summary extends React.Component {
           {/* summary-row-3 */}
           <div className={c(t.mt4, t.flex, t.justifyBetween, t.itemsCenter)}>
             <div className={c(t.flex)}>
+              <Link
+                styles={{root: [FontClassNames.mediumPlus]}}
+                href={getTensorBoardUrl(jobInfo, rawJobConfig)}
+                disabled={isNil(setTensorBoardDisabled(jobInfo, rawJobConfig))}
+                target="_blank"
+              >
+                Go to TensorBoard Page
```

Comment:
rename to `Go to TensorBoard`? could you place it at the end of the row?

Reply:
Got it. This button is named like the other buttons here, such as "Go to Application Tracking Page" and "Go to Metrics Page".